Packs/Campaign/ReleaseNotes/3_4_0.md

```diff
@@ -0,0 +1,52 @@
+
+#### Scripts
+
+##### GetCampaignIncidentsInfo
+
+- Documentation and metadata improvements.
+
+##### ShowCampaignIncidentsOwners
+
+- Fixed an issue where `Incident Owners` section raised an error when no incidents were present in the context data.
+
+##### ShowCampaignLastIncidentOccurred
+- Fixed an issue where `Last Incident Occurred` section raised an error when no incidents were present in the context data.
+
+##### ShowCampaignRecipients
+
+- Fixed an issue where `Campaign Recipients` section raised an error when no incidents were present in the context data.
+
+##### ShowCampaignSenders
+
+- Fixed an issue where `Campaign Senders` section raised an error when no incidents were present in the context data.
+
+##### ShowCampaignSimilarityRange
+
+- Fixed an issue where `Similarity Range` section raised an error when no incidents were present in the context data.
+
+##### GetCampaignIndicatorsByIncidentId
+- Updated the Docker image to: *demisto/pandas:1.0.0.83429*.
+
+- Fixed an issue where `Mutual Campaign Indicators` section raised an error when no incidents were present in the context data.
+
+
+##### PerformActionOnCampaignIncidents
+
+- **Braking Change:** Replaced the "Link" Button to "Add to campaign" Button to manually add incident(s) to a campaign using the incident id.
+- **Braking Change:** Replaced the "Unlink" Button to "Remove from Campaign" Button to remove manually incident(s) from a campaign using the incident id.
+- **Braking Change:** Replaced the "Link & Close" Button to "Add To Campaign & Close". This will add the incident to the campaign and close the incident.
```

Comment:
```suggestion
**Breaking Changes:**
- Replaced the "Link" button with the "Add to campaign" button to manually add incident(s) to a campaign using the incident ID.
- Replaced the "Unlink" button with the "Remove from Campaign" button to manually remove incident(s) from a campaign using the incident ID.
- Replaced the "Link & Close" button with the "Add To Campaign & Close" button. This will add the incident to the campaign and close the incident.
```